site/content/en/docs/manual/advanced/formats/_index.md

```diff
@@ -1,28 +1,103 @@
 weight: 20
-description: 'List of annotation formats supported by CVAT.'
+description: 'List of data export formats formats supported by CVAT.'
 ---
 
-#### CVAT supported the following formats:
-
-- [CVAT](format-cvat)
-- [Datumaro](format-datumaro)
-- [LabelMe](format-labelme)
-- [MOT](format-mot)
-- [MOTS](format-mots)
-- [COCO](format-coco)
-- [PASCAL VOC and mask](format-voc)
-- [YOLO](format-yolo)
-- [TF detection API](format-tfrecord)
-- [ImageNet](format-imagenet)
-- [CamVid](format-camvid)
-- [WIDER Face](format-widerface)
-- [VGGFace2](format-vggface2)
-- [Market-1501](format-market1501)
-- [ICDAR13/15](format-icdar)
-- [Open Images](format-openimages)
-- [Cityscapes](format-cityscapes)
-- [KITTI](format-kitti)
-- [LFW](format-lfw)
+In CVAT, you have the option to export data in various formats.
+The choice of export format depends on the type of annotation as
+well as the intended future use of the dataset.
+
+See:
+
+- [Data export formats](#data-export-formats)
+- [Exporting dataset in CVAT](#exporting-dataset-in-cvat)
+  - [Exporting dataset from Task](#exporting-dataset-from-task)
+  - [Exporting dataset from Job](#exporting-dataset-from-job)
+- [Data export video tutorial](#data-export-video-tutorial)
+
+## Data export formats
+
+The table below outlines the available formats for data export in CVAT.
+
+<!--lint disable maximum-line-length-->
+
+| Format                                                                                                                              | Type          | Annotation Type                                             | Models                                                                                                                                                                                  | Shapes                                                                                 | Attributes           | Video Tracks  |
+| ----------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------- | -------------------- | ------------- |
+| [CamVid 1.0](format-camvid)                                                                                                         | .txt <br>.png | Semantic <br>Segmentation                                   | U-Net, SegNet, DeepLab, <br>PSPNet, FCN, Mask R-CNN, <br>ICNet, ERFNet, HRNet, <br>V-Net, and others.                                                                                   | Bounding Boxes, Polygons                                                               | Not supported        | Not supported |
```

Comment:
Let's remove "shape == bounding box" for all datasets for segmentation only. It doesn't make sense. There are some datasets which natively support bounding boxes and segmentations. I believe CamVid, Cityscapes, etc are not in the list.